setup.py

```diff
@@ -184,5 +184,5 @@
  'packages': ['fqn_decorators'],
  'tests_require': ['tox'],
  'url': 'https://github.com/kpn-digital/py-fqn-decorators',
- 'version': '1.2.0',
+ 'version': '1.2.4',
```

Comment:
actually, this file is automatically generated(version is generated from git tag) and should be excluded from git. you don't need to change anything here.